terraform/stacks/tooling/outputs.tf

```diff
@@ -455,3 +455,8 @@ output "bosh_uaa_elb_dns_name" {
 output "bosh_uaa_elb_name" {
   value = "${aws_elb.bosh_uaa_elb.name}"
 }
+
+/* Static proxy */
+output "static_proxy_ip" {
```

Comment:
Do we want the name to be this generic? Is it possible that in the future we'd offer multiple EIPs? or would everyone just use the same one?